README.md

```diff
@@ -1,11 +1,2 @@
-# Cyber Discovery Rich Presence
```

Comment:
Have you actually checked the diff?

Reply:
i literally just downloaded it and made changes to the cs file